src/web/src/components/DepartmentList.vue

```diff
@@ -17,7 +17,41 @@
             role="tabpanel"
           >
             <b-card-body>
-              <b-card-text>TODO: Insert Major's Classes</b-card-text>
+              <div id="scroll-box">
+                <recycle-scroller
```

Comment:
I'd suggest using `<list>` and `<listitem>` here instead of `<recycle-scroller>`